app/models/updates_production_credits.rb

```diff
@@ -0,0 +1,54 @@
+class UpdatesProductionCredits
+  def self.update(generic_file)
+    new(generic_file).update
+  end
+
+  def initialize(generic_file)
+    @generic_file = generic_file
+  end
+
+  def update
+    update_productions
+    update_venues
+    update_works
+  end
+
+  private
+
+  attr_reader :generic_file
+
+  def update_productions
+    generic_file.production_names = productions.map(&:production_name).compact
```

Comment:
`productions.map(&:production_name).compact` could be pulled out into it's own method. `all_productions`? possibly on the model?